region/brinstar/kraid.json

```diff
@@ -468,8 +518,9 @@
           "to": [
             {
               "id": 1,
-              "requires": [ "canPassBombPassages" ],
-              "unlock": null
+              "requires": null,
+              "unlock": null,
+              "note": "Reaching 4 means obstacle B is open."
```

Comment:
I don't see the point of making that assumption. We can have the link require Morph + breaking the obstacle and it'll dynamically translate to just Morph.

Reply:
The point is to do less checks.

Reply:
Doesn't really matter I guess.